vsintegration/tests/FSharp.Editor.Tests/DocumentDiagnosticAnalyzerTests.fs

```diff
@@ -28,25 +28,25 @@ type DocumentDiagnosticAnalyzerTests() =
         let errors = getDiagnostics fileContents
 
         if not errors.IsEmpty then
-            Assert.Fail("There should be no errors generated", errors)
+            failwith $"There should be no errors generated: {errors}"
 
     member private this.VerifyErrorAtMarker(fileContents: string, expectedMarker: string, ?expectedMessage: string) =
         let errors =
             getDiagnostics fileContents
             |> Seq.filter (fun e -> e.Severity = DiagnosticSeverity.Error)
             |> Seq.toArray
 
-        Assert.AreEqual(1, errors.Length, "There should be exactly one error generated")
+        Assert.True(1 = errors.Length, "There should be exactly one error generated")
         let actualError = errors.[0]
 
         if expectedMessage.IsSome then
-            Assert.AreEqual(expectedMessage.Value, actualError.GetMessage(), "Error messages should match")
+            Assert.True(expectedMessage.Value = actualError.GetMessage(), "Error messages should match")
```

Comment:
Is there nothing like Assert.Equal in xunit? That should give a more useful error message in case the test fails (i.e. I hope it would include what the actual error message was)

Reply:
There is Assert.Equal(x,y), there is no builtin Assert.Equal(x,y,message).
But for that (and other things) we have those Assert extensions in FSharp.Test.Utilities project, those build on top of Fluent Assertions. I updated this, thanks for noticing